ubuntu/debian/libignition-transport.install

```diff
@@ -1 +1,4 @@
 usr/lib/*/libignition-transport[0-99].so.*
+usr/lib/ruby/ignition/cmdtransport[0-99].rb
```

Comment:
I think we need to remove these from `-code-dev`, otherwise if multiple packages install the same file they aren't side-by-side installable

https://github.com/gazebo-release/gz-transport8-release/blob/f6af6cf89281c714373fc43112a0c4e4a5aa92ea/ubuntu/debian/libignition-transport-core-dev.install#L8-L9